repose-aggregator/components/services/atom-feed-service/atom-feed-service-api/build.gradle

```diff
@@ -10,3 +10,7 @@ jacocoTestCoverageVerification {
         }
     }
 }
+
+dependencyCheck {
+    skip = true
```

Comment:
Why?

Reply:
I'm not actually sure on this one. @wdschei?